### PR TITLE
Pull RDN printing into a seperate template with fixes

### PR DESCRIPTION
**Pull request by mcpherrinm**
Print all the values that Go parses out, and print OIDs and values for the
ExtraNames Values that Go allows multiple values (everything other than CN) are
printed multiple times instead of in a square-bracket array.  Since most certs
have only a single value for each, this just avoids the square brackets.

### CODE REVIEW

File: lib/display.go [hunk 1]
```diff
@@ -29,23 +29,51 @@ import (
...
+{{- end -}}
+{{- range .StreetAddress}}
+	StreetAddress: {{.}}
+{{- end -}}
+{{- range .PostalCode}}
+	PostalCode: {{.}}
+{{- end -}}
+{{- range .ExtraNames -}}
+	{{- range $index, $type := .Type }}
+		{{- if $index }}.{{else}}
```
Comment (csstaub) [inline lib/display.go]: if $index > 0 print a period, else print a newline? Not sure I understand

Comment (mcpherrinm) [inline lib/display.go]: This is printing the OID.  So on the first entry, we print \n\t, and then seperate with dots

So we end up with something that looks like
```
        0.1.2.3: Foo
        5.6.7345.9: Bar
```

I should include a test case for this -- I'll add one

File: lib/display.go [hunk 2]
```diff
@@ -246,3 +221,26 @@ func certEnd(end time.Time) string {
 func redify(text string) string {
 	return red.SprintfFunc()("%s", text)
 }
+
+func oidify(oid asn1.ObjectIdentifier) string {
+	raw := oid.String()
+	names := map[string]string{
+		"2.5.4.3":                  "CommonName",
+		"2.5.4.5":                  "Serial Number",
```
Comment (csstaub) [inline lib/display.go]: Change this to Registration Number to match the language in the CAB guidelines?

File: lib/display.go [hunk 3]
```diff
@@ -246,3 +221,26 @@ func certEnd(end time.Time) string {
...
+		"2.5.4.6":                  "Country",
+		"2.5.4.7":                  "Locality",
+		"2.5.4.8":                  "Province",
+		"2.5.4.10":                 "Organization",
+		"2.5.4.11":                 "OrganizationalUnit",
+		"2.5.4.15":                 "Business Category",
+		"1.2.840.113549.1.9.1":     "emailAddress",
+		"1.3.6.1.4.1.311.60.2.1.1": "EV Locality",
+		"1.3.6.1.4.1.311.60.2.1.2": "EV Province",
+		"1.3.6.1.4.1.311.60.2.1.3": "EV Country",
```
Comment (csstaub) [inline lib/display.go]: I'd call this "Inc. Country" etc. to match Chrome/browser stuff.

File: lib/display.go [hunk 4]
```diff
@@ -246,3 +221,26 @@ func certEnd(end time.Time) string {
...
+func oidify(oid asn1.ObjectIdentifier) string {
+	raw := oid.String()
+	names := map[string]string{
+		"2.5.4.3":                  "CommonName",
+		"2.5.4.5":                  "Serial Number",
+		"2.5.4.6":                  "Country",
+		"2.5.4.7":                  "Locality",
+		"2.5.4.8":                  "Province",
+		"2.5.4.10":                 "Organization",
+		"2.5.4.11":                 "OrganizationalUnit",
```
Comment (csstaub) [inline lib/display.go]: Space, like in other fields? "Organizational Unit"

Comment (mcpherrinm) [inline lib/display.go]: sure

File: lib/display.go (hunk 2 above)
Comment (mcpherrinm) [inline lib/display.go]: I think that seems fine - it doesn't really make sense for non-EV certs, but I don't think I've ever seen 2.5.4.5 used otherwise.

**Comment by mcpherrinm** [comment]
I did the same changes to the JSON output.

**Pull request merged**